modules/ml/src/main/java/org/apache/ignite/ml/math/primitives/vector/VectorUtils.java

```diff
@@ -50,15 +50,35 @@ public static DenseVector fill(double val, int n) {
         return (DenseVector)new DenseVector(n).assign(val);
     }
 
+    /**
+     * Wrap specified value into vector.
+     *
+     * @param val Value to wrap.
+     * @return Specified value wrapped into vector.
+     */
+    public static Vector num2Vec(double val) {
```

Comment:
If you don't mind, I wouldn't change it now because otherwise for consistency I should also change other such methods. I think it should be done atomically after some discussion.